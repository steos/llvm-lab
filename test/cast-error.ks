
	void main()
	{
		int foo = true;
		iprint(foo);
	}
	
	native void iprint(int);