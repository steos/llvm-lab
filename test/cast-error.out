cannot cast boolean to numeric type on line 4 at offset 10
